.github/workflows/test.yml

```diff
@@ -64,6 +64,13 @@ jobs:
     - name: Unit Tests
       run: make test-unit
 
+    - name: Post test results to New Relic
+      uses: newrelic/junit-reporter-action@v1
```

Comment:
You might need to use `newrelic/junit-reporter-action@master` here since it's **v0.1.0**. Either that or make an official `v1.0.0` release of that GH action. Unfortunately I don't think the actions framework resolves minor versions the way we might expect.

Reply:
Thanks!